### PR TITLE
approf 14.1.1 (new cask)

### PR DESCRIPTION
**Pull request by clement2026**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by clement2026** [comment]
> Error: GitHub repository too new (<30 days old)

Retry on September 6th.

File: Casks/a/approf.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  url "https://github.com/moderato-app/approf/releases/download/v#{version}/approf-#{version}.app.zip"
+  name "approf"
+  desc "Native app for pprof"
+  homepage "https://github.com/moderato-app/approf"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
```
Comment (p-linnane) [inline Casks/a/approf.rb:14]:
```suggestion

```

Comment (p-linnane) [reply Casks/a/approf.rb:14]: This livecheck can be removed. The git tags will be checked by default here.

**Review by p-linnane — COMMENTED** [review]
I've added a label to skip the repository check. Please address my other comment and I'm fine with merging this.

**Comment by clement2026** [comment]
> I've added a label to skip the repository check. Please address my other comment and I'm fine with merging this.

Thanks a lot! This is super helpful.

**Review by p-linnane — APPROVED** [review]
Thanks!

**Pull request merged**